modules/ROOT/pages/hl7/hl7-connector.adoc

```diff
@@ -664,7 +681,7 @@ The HL7 message contains the following keys:
 [%header%autowidth.spread]
 |===
 |Key name |Description
-|ACK (read only) |ACK message generated in response to the input data. The MSA-01 acknowledgment code value is based on the parser configuration settings. To send an acknowledgment, see the Sending Acknowledgments section below.
+|ACK (read only) |ACK message generated in response to the input data. The MSA-01 acknowledgment code value is based on the parser configuration settings. To send an acknowledgment, see the Sending Acknowledgments section.
```

Comment:
Maybe link to the Sending Acknowledgements section.

Reply:
Not at this time, wanna get this out.